### ISSUE TITLE
Missing E701 when the physical line contains multiple colons

### ISSUE THREAD

**Issue by florentx** [PyCQA/pycodestyle#196]
This example should trigger E701:

``` python
def parse_header(header):
    if not header or header[:6] != 'bytes=': return
    # ...
```
